Makefile

```diff
@@ -0,0 +1,25 @@
+.PHONY: test test-verbose test-profile clean docker bootstrap
+
+PACKAGES = `go list ./... | grep -v vendor/`
+
+all: inertia
+
+inertia:
+	go build
+
+test:
+	go test $(PACKAGES) --cover
+
+test-verbose:
+	go test $(PACKAGES) -v --cover
+
+clean: inertia
+	rm -f inertia
+	rm -f cover.html
+
+docker:
+	docker build -t ubclaunchpad/inertia .
+	docker push ubclaunchpad/inertia
+
+bootstrap:
+	go-bindata -o client/bootstrap.go client/bootstrap/...
```

Comment:
+1. we should also either make the bootstrap file either a part of the main package or use a python script to replace the package header inside the bootstrap file after we automatically generate it. I’m partial to the former.

Reply:
I don't think we can put it in `main` since that would be a cyclic dependency if `client` uses it right?

Reply:
Yeah exactly, I think that’s why I didn’t do it in the first place 🤔 maybe there’s a flag we can use to set the package name?

Reply:
It's [somewhere](https://github.com/jteeuwen/go-bindata/blob/master/config.go#L28)